addons/escoria-core/game/core-scripts/esc/commands/camera_push.gd

```diff
@@ -13,9 +15,9 @@
 # Supported transitions include the names of the values used
 # in the "TransitionType" enum of the "Tween" type (without the "TRANS_" prefix):
 #
-# https://docs.godotengine.org/en/stable/classes/class_tween.html?highlight=tween#enumerations
+# See https://docs.godotengine.org/en/stable/classes/class_tween.html?highlight=tween#enumerations
 #
-# For more details see: https://docs.escoria-framework.org/camera
+# For more details see: https://docs.escoria-framework.org/en/devel/getting_started/camera.html
```

Comment:
@dploeger (Comment from @BHSDuncan on original PR) : will this tie us into this particular link always using the devel branch of the docs? What if we eventually deploy from a different branch? Or am I way off?

There are a few references pointing to this development branch link across this PR that may need to be corrected based on your response.

Reply:
Yes, the way here is to go with redirects. /camera should be a redirect to the develop page currently, but apparently is missing.

That's a thing we can configure on the read the docs admin page.

Can you kindly filter out the revelant links and list here? I'll add the redirects then.

Reply:
@dploeger Here's all the links listed based off a grep in every .gd file

https://docs.escoria-framework.org/camera
https://docs.escoria-framework.org/en/devel/getting_started/camera.html
https://docs.escoria-framework.org/states for details.
https://docs.godotengine.org/en/stable/classes/class_string.html#class-string-method-match)
https://docs.godotengine.org/en/stable/classes/class_tween.html#enumerations
https://docs.godotengine.org/en/stable/classes/class_tween.html?highlight=tween#enumerations

Reply:
- [ ] https://docs.escoria-framework.org/camera
- [ ] https://docs.escoria-framework.org/en/devel/getting_started/camera.html
- [ ] https://docs.escoria-framework.org/states
- [ ] https://docs.godotengine.org/en/stable/classes/class_string.html#class-string-method-match
- [ ] https://docs.godotengine.org/en/stable/classes/class_tween.html#enumerations
- [ ] https://docs.godotengine.org/en/stable/classes/class_tween.html?highlight=tween#enumerations)

Reply:
Okay, that's not much. You can use https://docs.escoria-framework.org/camera for https://docs.escoria-framework.org/en/devel/getting_started/camera.html.

I just added the redirect.

The godot docs are outside of our jurisdiction, so I'd just keep the pages under docs.godotengine.org as is.

Reply:
Thanks @dploeger - I've corrected all the camera references accordingly (I'll squash the commits at a later point)